pydoctor/zopeinterface.py

```diff
@@ -221,13 +220,19 @@ def _handleAssignmentInClass(self,
                     section='zopeinterface')
         else:
             match = schema_prog.match(funcName)
+
             if match:
-                attr.kind = match.group(1)
+                attr.kind = model.KindClass.SCHEMA_FIELD
+
             else:
                 cls = self.builder.system.objForFullName(funcName)
                 if not (isinstance(cls, ZopeInterfaceClass) and cls.isschemafield):
                     return
-                attr.kind = cls.name
+                attr.kind = model.KindClass.SCHEMA_FIELD
+
+            # Linking to zope.schema.* class if setup with intersphinx
```

Comment:
```suggestion
            # Link to zope.schema.* class, if setup with intersphinx.
```
It's nice that we get slightly richer output now, by linking to the zope docs.